src/traces/indicator/attributes.js

```diff
@@ -0,0 +1,408 @@
+        description: [
+            'Sets the range of this axis.',
+            'If the axis `type` is *log*, then you must take the log of your',
+            'desired range (e.g. to set the range from 1 to 100,',
+            'set the range from 0 to 2).',
+            'If the axis `type` is *date*, it should be date strings,',
+            'like date data, though Date objects and unix milliseconds',
+            'will be accepted and converted to strings.',
+            'If the axis `type` is *category*, it should be numbers,',
+            'using the scale where each category is assigned a serial',
+            'number from zero in the order it appears.'
+        ].join(' ')
+    }
+}));
+
+module.exports = {
+    mode: {
+        valType: 'flaglist',
+        editType: 'calc',
+        role: 'info',
+        flags: ['number', 'delta', 'gauge'],
+        dflt: 'number'
+    },
+    value: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        anim: true,
+        description: [
+            'Sets the number to be displayed.'
+        ].join(' ')
+    },
+    align: {
+        valType: 'enumerated',
+        values: ['left', 'center', 'right'],
+        role: 'info',
+        editType: 'plot',
+        description: [
+            'Sets the horizontal alignment of the `text` within the box.',
+            'Note that this attribute has no effect if an angular gauge is displayed:',
+            'in this case, it is always centered'
+        ].join(' ')
+    },
+    // position
+    domain: domainAttrs({name: 'indicator', trace: true, editType: 'calc'}),
+
+    vmin: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        dflt: 0,
+        description: [
+            'Sets the minimum value of the gauge.'
+        ].join(' ')
+    },
+    vmax: {
+        valType: 'number',
+        editType: 'calc',
+        role: 'info',
+        description: [
+            'Sets the maximum value of the gauge.'
+        ].join(' ')
+    },
+
+    title: {
+        text: {
+            valType: 'string',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the title of this indicator.'
+            ].join(' ')
+        },
+        align: {
+            valType: 'enumerated',
+            values: ['left', 'center', 'right'],
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the horizontal alignment of the title.',
+                'It defaults to `center` except for bullet charts',
+                'for which it defaults to right.'
+            ].join(' ')
+        },
+        font: extendFlat({}, textFontAttrs, {
+            description: [
+                'Set the font used to display the title'
+            ].join(' ')
+        }),
+        editType: 'plot'
+    },
+    number: {
+        valueformat: {
+            valType: 'string',
+            dflt: '.3s',
```

Comment:
It might be nice to be consistent with other `*format` attributes in use currently [e.g. `tickformat` for axis](https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/src/plots/cartesian/layout_attributes.js#L549-L564) where the `dflt` is left blank and we let [`Axes.tickText`](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/axes.js#L934-L1004) handle things.

Reply:
I don't have a strong opinion for this one but having a fixed number of digits with SI suffixes as a default does make sense if we want the number to always fit within the angular gauge regardless of how big or small the value is :thinking:

Reply:
> for this one but having a fixed number of digits with SI suffixes as a default does make sense if we want the number to always fit within the angular gauge regardless of how big or small the value is

Good point here. I don't have a strong opinion either.